### ISSUE TITLE
[Feature] + also some BUGS! Media gallery/upload: combined new ticket

### ISSUE THREAD

**Issue by kaatvds** [massenergize/frontend-admin#1034]
We had a lot of tickets on this and lost track of it. am harvesting from those tickets (https://github.com/massenergize/frontend-admin/issues/712 and https://github.com/massenergize/frontend-admin/issues/665) and closing them. Starting a new ticket. 

We turned off the Gallery (https://github.com/massenergize/frontend-admin/issues/794)  and for the moment the only way to see any images in the DB is through the various "insert image" buttons for actions etc. I think that is fine for now and am wondering i we need a sepate Media Gallery at all - although, I am also interested in how such a media gallery could operate with our media center...

BUG on prod: When one clicks on "Add filters to tue your search" the screen goes blank: 
<img width="1461" alt="Screenshot 2023-06-29 at 9 25 20 AM" src="https://github.com/massenergize/frontend-admin/assets/30277868/fafdc1b7-3ae3-4421-bb2d-bf2feb5cf46b">

**When uploading:**
1. name of admin (auto)
2. name of community (auto) unless, If admin is admin of more than one community, ask: what community are you uploading this for? Give dropdown of all their communitiies and they select one (we just need the first one)
3. Date (auto)
4. ask them to supply 5 keywords from a list: QUESTION: how to ask? how display? type in a word and it pops up, or a limited list?  etc. To explore. Can also be a combination with some top tags e.g. "graphic, photo, logo."  If it's a logo of a vendor or a recongizable building, etc, ask for the proper name.
5. what this image is used for: vendor, action, event (auto)
7. who they want to share this with: just their own community - and give examples of why (e.g., MY people), a subset, or all?
8.  ask them to tick a box stating explicitly that they have permission to use this image. Reminder them about their MOU. 
9. a question: does this picture recognizably show children under the age of ??? 13?? if so,  tick this box to state that you have written and recorded permission from guardians (and give them opportunity to type in some info but that is their responsiblity). If they tick this, the image (and the note) should automatically be restricted to just their community.

**For each image: display info:** 
In the side panel when selecting image, display this info for each image (added autom or added by admin, cf above)
1. name of admin who uploaded : call them the "owner"
2. name of the community that admin belongs to ("source community"). If admin is admin of more than one, they should be asked at time of upload what community they are uploading this for (cf above)
3. date of upload? 
4. keywords
5. what this image is used for: vendor, action, event  ("category")
6. N/A
7. permissions
8. special copyright notes (for images in admin's own community -  cf.above)

**search/filter**

1.  by "owner"
2. by community
3. NA
4. by keyword search
5. by "category"

**Order/Sort** images should not just show up in an arbitrary order.: so how are they ordered?
1. by date

**Comment by BradHN1** [comment]
Also please make the gallery accessible to Community Admins, but with a FeatureFlag to enable it in certain communities until we get the functionality right.

**Comment by kaatvds** [comment]
flagging that this ticket is still very much undercevelopmen. Have moved speccing [to this doc.](https://docs.google.com/document/d/15W2CmURenZjPxKq71nCuw2gP-nljN3Q7hWhQj1EEdzs/edit)

**Comment by frimpongopoku** [comment]
@kaatvds just some tiny clarification questions. 

1. In "When uploading" #2. You indicated that users should be able to select a community from their list of communities. And they should select "only One". Are you saying, we should limit the number of communities that an image can be available to.. to only 1? 
2. I also assumed admins would be able to indicate that their images should be available to any other communities outside of the communities they manage. Are you saying we should restrict that and only allow admins to be able to post images around the few communities they manage? 

3. Also, in the part where admins are meant to indicate whether there are children in their photo, you wrote that if an admin ticks "yes' that there are children, the image should be restricted to only their community. Is this any different from the initial community target selection in "when uploading" #2 ?

**Comment by BradHN1** [comment]
Here are a couple of issues I noticed in canary testing, and if possible they would be fixed in the media library upgrade:

I submitted an event from the user portal, including a photo which I cropped (but the cropping didn't seem to work).  Separate issue, will submit to user-portal.

Got the email as a cadmin to review the event.  Since the photo isn't correct I want to crop the photo to make it a bit better.  I go to the media library, and the photo is there - Twice (Issue # 1)
There is no option to edit the photo to crop it (Issue #2)
![Screenshot 2023-08-09 at 11 21 21 AM](https://github.com/massenergize/frontend-admin/assets/12800484/732140a0-839c-4ebe-93b1-f14d17099573)
